.gitignore

```diff
@@ -46,6 +46,15 @@ __init__.py
 !setup/etc/profile.d/*sh
 !setup/etc/yum.repos.d/*.repo
 
+# Don't ignore gemonlinedb definition files
+!gemonlinedb/defs/*.json
+
+# Don't ignore gemonlinedb common xsd definitions
+!gemonlinedb/schema/common.xsd
+
+# Do ignore gemonlinedb auto-generated files
+gemonlinedb/**/*Gen.[h,cpp]
+
```

Comment:
It is probably more appropriate to put these `gemonlinedb` specific items in, e.g., `gemonlinedb/.gitignore`